src/concrete_convex_hull.jl

```diff
@@ -136,6 +119,48 @@ function convex_hull!(points::Vector{VN};
     end
 end
 
+function _two_points_2d!(points)
+
+    n = length(first(points))
+
+    if n == 1
```

Comment:
Remove the case split on `n` because the function name suggests that you already checked that the points are two-dimensional.